x-pack/legacy/plugins/transform/server/routes/api/app.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Router, RouterRouteHandler } from '../../../../../server/lib/create_router';
+import { wrapCustomError } from '../../../../../server/lib/create_router/error_wrappers';
+import { APP_CLUSTER_PRIVILEGES, APP_INDEX_PRIVILEGES } from '../../../common/constants';
+// NOTE: now we import it from our "public" folder, but when the Authorisation lib
+// will move to the "es_ui_shared" plugin, it will be imported from its "static" folder
+import { Privileges } from '../../../public/app/lib/authorization';
+import { Plugins } from '../../shim';
+
+let xpackMainPlugin: any;
+
+export function registerAppRoutes(router: Router, plugins: Plugins) {
+  xpackMainPlugin = plugins.xpack_main;
```

Comment:
You should be able to pull licensing information from the New Platform licensing plugin rather than the legacy plugin. It's available via `server.newPlatform.setup.plugins.licensing`

Reply:
Thanks, I looked into that a bit now but will follow up in another PR. I added a note in #45352.